### ISSUE TITLE
Create initial version of IDE page

### ISSUE THREAD

**Issue by fabianmuehlboeck** [Tateology/kaeru#32]
@Hapa29 

According to #6

**Comment by Hapa29** [comment]
Here's what I have so far:

![ideversion01](https://cloud.githubusercontent.com/assets/3515686/7195426/57e559de-e48d-11e4-80b7-b6426705ab4b.png)

The basic layout is there, but nothing is functional and none of the sections are collapsible or resizable yet. Do we want each section to be resizable and/or collapsible?

**Comment by fabianmuehlboeck** [comment]
Yes, if possible. The file part may not absolutely need to be resizable, but it might come handy in some situations, so I guess that depends on how hard it is to make it happen.
